plugins/package-managers/gleam/src/main/kotlin/GleamToml.kt

```diff
@@ -75,6 +73,10 @@ internal data class GleamToml(
     /** Links to project resources like homepage, documentation, etc. */
     val links: List<Link> = emptyList()
 ) {
+    companion object {
+        private val HOMEPAGE_KEYS = listOf("website", "home page", "homepage")
+    }
```

Comment:
I believe this is slightly inconsistent with the majority in this code base. I'd argue that
there is not much value in this scope reduction, and the benefit normally added by companion object of "discoverability" is neglectible, because the scope is private. Therefore, I think we should not use the companion object in this case. What do you think?